java/kotlin.editor/src/org/netbeans/modules/kotlin/editor/layer.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
(Assuming this is still needed:) It would probably be better to enhance the existing annotation-based declarations, rather than using an explicit layer.

Reply:
I've finally find out how to do that. Thank you!